client/omnichannel/directory/ContactInfo.js

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from 'react';
+import { Box, Margins, ButtonGroup, Button, Icon } from '@rocket.chat/fuselage';
+import { css } from '@rocket.chat/css-in-js';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+
+import VerticalBar from '../../components/VerticalBar';
+import UserCard from '../../components/UserCard';
+import { FormSkeleton } from './Skeleton';
+import { useEndpointData } from '../../hooks/useEndpointData';
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useRoute } from '../../contexts/RouterContext';
+import { hasPermission } from '../../../app/authorization';
+import { useFormatDate } from '../../hooks/useFormatDate';
+import UserAvatar from '../../components/avatar/UserAvatar';
+import { UserStatus } from '../../components/UserStatus';
+import { AsyncStatePhase } from '../../hooks/useAsyncState';
+
+const wordBreak = css`
+	word-break: break-word;
+`;
+const Label = (props) => <Box fontScale='p2' color='default' {...props} />;
+const Info = ({ className, ...props }) => <UserCard.Info className={[className, wordBreak]} flexShrink={0} {...props}/>;
+
+function ContactManagerField({ username }) {
```

Comment:
`Contact Manager` is an Enterprise Feature, which means it needs to be placed inside the `EE` folder, the same way we do with `DepartmentForwarding` form.